### PR TITLE
Issue and PR templates

### PR DESCRIPTION
**Pull request by emsesc**
Added templates to .github

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/test-bug.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: "🐛 Test Bug Report"
```
Comment (blueboxes) [inline .github/ISSUE_TEMPLATE/test-bug.md:2]: Should we change to "🧪 Test Code Bug Report" to show it is not Testing the Bug Reporting?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Changes made
+* [Replace with summaries of general changes made]
+* [Did you modify or delete?]
+* [How do your changes fix the issue?]
+
+Closes #[issue number]
```
Comment (blueboxes) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: Should be [Issue Link] I think so GitHub auto links it.

**Review by blueboxes — CHANGES_REQUESTED** [review]
You have "🐛 Test Bug Report" and "⭐️ Feature or Content Request" what about additions/enhancement to the tests?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (blueboxes) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Ignore me on this one what you have will auto-link https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by emsesc** [comment]
> You have "🐛 Test Bug Report" and "⭐️ Feature or Content Request" what about additions/enhancement to the tests?

@blueboxes good point, do you think it's plausible to have additions to tests be considered as a feature addition?

**Comment by blueboxes** [comment]
The templates are all about the questions. I think the new feature/content template would work fin maybe just also included improvement to test in the description.

**Review by blueboxes — APPROVED** [review]
All looks good 👍

**Pull request merged**